apps/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -202,29 +248,51 @@ export class CommandLineConfiguration {
         let normalizedCommand: Command;
         switch (command.commandKind) {
           case RushConstants.phasedCommandKind: {
+            const commandPhases: Set<IPhase> = new Set();
+
             normalizedCommand = {
               ...command,
               isSynthetic: false,
-              associatedParameters: new Set<Parameter>()
+              associatedParameters: new Set<Parameter>(),
+              phases: commandPhases
             };
 
-            for (const phaseName of normalizedCommand.phases) {
-              if (!this.phases.has(phaseName)) {
+            for (const phaseName of command.phases) {
+              const phase: IPhase | undefined = this.phases.get(phaseName);
+              if (!phase) {
                 throw new Error(
                   `In ${RushConstants.commandLineFilename}, in the "phases" property of the ` +
                     `"${normalizedCommand.name}" command, the phase "${phaseName}" does not exist.`
                 );
               }
+
+              commandPhases.add(phase);
+            }
+
+            // Apply implicit phase dependency expansion
+            // The equivalent of the "--to" operator used for projects
+            for (const phase of commandPhases) {
```

Comment:
Recursive expansion?

Reply:
This does a full breadth-first search. Updated the comment. Same algorithm is used to apply the `--to` operator in `Selection.expandAllDependencies`